modules/core/src/main/java/org/apache/ignite/internal/visor/consistency/VisorConsistencyRepairTask.java

```diff
@@ -198,6 +162,8 @@ protected VisorConsistencyRepairJob(VisorConsistencyRepairTaskArg arg, boolean d
 
                             log.info("Consistency check progress [grp=" + grpCtx.cacheOrGroupName() +
                                 ", part=" + p + ", checked=" + cnt + "/" + part.fullSize() + "]");
+
+                            VisorConsistencyStatusTask.MAP.put(arg, cnt + "/" + part.fullSize());
```

Comment:
Should the map also be notified a little bit earlier? See line 114 with logging "Consistency check started"? I think we should put to the map (arg, 0/part.fullSize()) in this moment, WDYT?

Otherwise the status job may return that no repair jobs are found, but actually there are in progress.